src/main/java/com/google/plantasy/servlets/UserQuizStatusServlet.java

```diff
@@ -1,32 +1,29 @@
 package com.google.plantasy.servlets;
- 
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
 import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.appengine.api.datastore.FetchOptions;
-import java.text.DateFormat;
-import java.util.Date;
-import com.google.appengine.api.datastore.Key;
-import com.google.plantasy.QuizTimingPropertiesUtils;
-import com.google.plantasy.UserUtils;
- 
+import com.google.plantasy.utils.QuizTimingPropertiesUtils;
+import com.google.plantasy.utils.UserUtils;
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
```

Comment:
please import specifically what you need rather than *

Reply:
Fixed this!